jupyter-extension/jupyterlab_pachyderm/mount_server_client.py

```diff
@@ -72,8 +89,9 @@ async def _ensure_mount_server(self):
         async with lock:
             if not await self._is_mount_server_running():
                 self._unmount()
-
-                mount_server_cmd = f"mount-server --mount-dir {self.mount_dir}"
+                mount_server_cmd = f"mount-server --mount-dir {self.mount_dir} --sock-path {self.sock_path}"
+                if self.sock_path == "":
+                    mount_server_cmd = f"mount-server --mount-dir {self.mount_dir}"
```

Comment:
This is minor, but you could do the following to remove the duplicated code:
```python
mount_server_cmd = f"mount-server --mount-dir {self.mount_dir}"
if self.sock_path:
    mount_server_cmd += f" --sock-path {self.sock_path}"
```

Reply:
ok

Reply:
done